pom.xml

```diff
@@ -9,23 +9,24 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.2.6.RELEASE</version>
+        <version>2.5.2</version>
     </parent>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <javafx.version>18-ea+1</javafx.version>
```

Comment:
why upgraded to an early access release? Any definite benefit?

Reply:
I am getting performance benefit by using this version, at least on my machine. With version 14, the app was very slow and was taking a lot of time even on repositioning on the screen.

Reply:
Cool. For me also the app is very slow on office laptop. Hope this fixes it.